src/main/java/frc/robot/util/ConversionUtils.java

```diff
@@ -81,6 +81,14 @@ public static double RPMToFalcon(double RPM, double gearRatio) {
         return motorRPM * (2048.0 / 600.0);
     }
 
+    public static double RPMToMeters(double RPM, double circumference, double gearRatio){
+        return falconToMeters(RPMToFalcon(RPM,gearRatio),circumference,gearRatio);
+    }
+
+    public static double RPMToMPS(double RPM, double gearRatio,double circumference){
+        return falconToMPS(RPMToFalcon(RPM, gearRatio),circumference,gearRatio);
+    }
```

Comment:
This method also probably should use a gear ratio of 1 for the inside function.

Reply:
Deleted it.